examples/profiling/index.js

```diff
@@ -0,0 +1,38 @@
+const { start, stop } = require('@splunk/otel');
+const { diag, DiagConsoleLogger, DiagLogLevel, trace, SpanStatusCode, context } = require('@opentelemetry/api');
+
+// If OTEL_LOG_LEVEL env var is set, configure logger
+if (process.env.OTEL_LOG_LEVEL) {
+  diag.setLogger(new DiagConsoleLogger(), DiagLogLevel[process.env.OTEL_LOG_LEVEL]);
+}
+
+// Profiling is still experimental and has to be enabled explicitly
+start({
+	profiling: {
+		callstackInterval: 100,
+		collectionDuration: 1_000,
```

Comment:
If it's example code then we're really omitting a lot of information here (service name, resource attributes, endpoint) as they come from the test's env file in a separate directory.

Besides by default we are currently recommended to use a call stack interval of 1000, both of these parameters go into the advanced usage/tweaks category.

Reply:
So what's your suggestion?

Choose to add those options because they are informative(specific to profiling) and make the example run faster. 30s might make the user question whether it's broken or not.

Perhaps I should add a note about the default being recommended for prod?

Reply:
Another option would be to move this file to the test dir itself?

Reply:
Would the motivation mainly be to remove an example that showcases the usage of unusual options or something else? We don't have the general options available in every example anyway. User has to provide a API token at minimum anyway. Resource attributes and endpoint config is shown in other examples - only makes it more confusing to include it in all of them.

We'd lose the example and have to change how the e2e tests are built. Imho not worth it.

Reply:
The comment about `setTimeout` is suboptimal for now, but it keeps us honest and we'll fix it tomorrow.

Reply:
At least a small README would be helpful in this case, as is in https://github.com/signalfx/splunk-otel-js/tree/main/examples/basic

Reply:
Yes, that makes sense. Added the readme